.travis.yml

```diff
@@ -3,6 +3,7 @@ rvm:
   - 1.9.3
 env: facebook_public_key=test facebook_private_key=test google_public_key=test google_private_key=test
 script:
+  - RAILS_ENV=test bundle exec rake db:migrate --trace
```

Comment:
I forgot what the error was specifically, but something about running the db:migrate command. I think it was similar to here: http://stackoverflow.com/questions/10591599/rake-dbmigration-not-working-on-travis-ci-build

@nalnaji